components/automate-backend-elasticsidecar/habitat/automate-backend-elasticsidecar/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
```

Comment:
We do not maintain Changelog in automate

Reply:
okay removing it